nx_parallel/__init__.py

```diff
@@ -1,5 +1,6 @@
+from .utils.decorators import _configure_if_nx_active
```

Comment:
As discussed in the last meeting, here I've added the `_configure_if_nx_active` in the nx_parallel's public API, mimicking how networkx adds the `_dispatchable` decorator in the networkx's public API by explicitly importing it in the `__init__.py` file